### PR TITLE
Mark Matchable::attribute_value() const

### PR DESCRIPTION
**Pull request by Minoru**
Just a small const correctness fix, I don't think it needs more justification than that :)

Reviews are welcome! I'll merge this in three days.

### CODE REVIEW

File: src/cache.cpp [hunk 1]
```diff
@@ -1100,7 +1100,7 @@ void Cache::fetch_descriptions(RssFeed* feed)
 	run_sql(query, fill_content_callback, feed);
 }
 
-std::string Cache::fetch_description(RssItem* item)
+std::string Cache::fetch_description(const RssItem* item)
```
Comment (dennisschagt) [inline src/cache.cpp:1103]: It might be useful to mark `RssItem` as a const reference because there is no checking for NULL.

**Review by dennisschagt — APPROVED** [review]
LGTM

File: src/cache.cpp (hunk 1 above)
Comment (Minoru) [reply src/cache.cpp]: Definitely! Done in 3a9dc0a.

**Review by dennisschagt — APPROVED** [review]
:+1:

**Pull request merged**